api/python/provisioner/commands/confstore_export.py

```diff
@@ -98,12 +117,23 @@ def run(self, **kwargs):
 
             Conf.load('provisioner', pillar[PillarKey(pillar_key)])
 
-            for data in template_data:
+            delimiter = '=>'
```

Comment:
I think it is better to put `'=>'` and `'>'` to constants. Why do we use these characters as delimiters? Based on the key representation used in consul for keys we can use `/`-character as keys delimiter.

Reply:
From Sradhanand
>This '>' delimiter is used by our confstore.
right now we are not using consul's confstore.
we are using the confstore which foundation team has created.

Reply:
>I think it is better to put '=>' and '>' to constants. 

Yes i will address this against EOS-19150.

>Why do we use these characters as delimiters? Based on the key representation used in consul for keys we can use /-character as keys delimiter.

currently we are not using consul as our confstore. The confstore we are using has some specific delimiters which they have specified.